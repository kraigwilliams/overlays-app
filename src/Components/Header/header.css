header{
    display:flex;
    align-items: center;
    justify-content: center;
    background:#1E555C;
    padding-top:20px;
    
}
#header, .header-color{
    background:#1E555C ;
 color:white;
    font-family: 'Coda Caption', sans-serif;
}

#nav{
    background:#1E555C;
 margin-left:30%;
 margin-right:30%;
align-self: flex-end;
}



@media  only screen and(min-width:768px){



}