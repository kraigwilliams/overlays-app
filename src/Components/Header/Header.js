import React from 'react';
import {Link} from 'react-router-dom';
import './header.css'

import Nav from '../Nav/Nav'

/*
Header Component for the entire App
*/ 
function Header(){
return(
    <header>
    <Link to= "/"><h1 id="header">Overlays App</h1></Link>
     <span id ="nav"><Nav /></span> 
    </header>
)

}


export default Header