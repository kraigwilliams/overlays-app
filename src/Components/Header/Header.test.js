import React from 'react';
//import { render } from '@testing-library/react';
import Header from './Header'
import {BrowserRouter} from 'react-router-dom'
import ReactDOM from  'react-dom';
import renderer from 'react-test-renderer';

describe('Header />',()=>{
  it ('is able to render',()=>{
    const div = document.createElement('div')
    ReactDOM.render(<BrowserRouter><Header/></BrowserRouter>, div)
    ReactDOM.unmountComponentAtNode(div)
  })

  it('renders the UI as expected', () => {
    const tree = renderer
      .create(<BrowserRouter><Header /></BrowserRouter>)
      .toJSON();
    expect(tree).toMatchSnapshot();  
    });

  
})