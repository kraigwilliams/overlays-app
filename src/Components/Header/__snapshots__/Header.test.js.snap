// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header /> renders the UI as expected 1`] = `
<header>
  <a
    href="/"
    onClick={[Function]}
  >
    <h1
      id="header"
    >
      Overlays App
    </h1>
  </a>
  <a
    href="/logout"
    id="nav-color"
    onClick={[Function]}
  >
    Login / Logout
  </a>
</header>
`;
